custom_metrics/third-parties.js

```diff
@@ -0,0 +1,34 @@
+[third-parties]
+/**
+ * Detect the presence of third party libraries.
+ *
+ * Look for common library aliases on the global scope.
+ * If available, detect libraries' versions.
+ *
+ * Outputs JSON-serialized list of library-version pairs.
+ *     e.g.: `["a@1.0","b","c@2.2"]`
+ */
+var thirdParties = [];
+var name, version;
+
+function addThirdParty(name, version) {
+	if (!name) {
+		return;
+	}
+
+	if (version) {
+		name += '@' + version;
+	}
+
+	thirdParties.push(name);
+}
+
+// jQuery
+var jQuery = window.jQuery || window.$ || window.$jq || window.$j;
+name = jQuery && jQuery.fn && 'jQuery';
+version = name && jQuery.fn.jquery
```

Comment:
That's how the block definition works for WebPageTest when submitting through the UI or API but not for permanent ones installed on the server.  Those should just be scripts with one script per metric.

i.e. yes, it should be removed.